src/test/java/org/jabref/logic/ai/chathistory/BibDatabaseChatHistoryManagerTest.java

```diff
@@ -1,50 +0,0 @@
-package org.jabref.logic.ai.chathistory;
-
-import java.nio.file.Path;
-import java.util.List;
-
-import dev.langchain4j.data.message.AiMessage;
-import org.h2.mvstore.MVStore;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.io.TempDir;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
-class BibDatabaseChatHistoryManagerTest {
```

Comment:
Is there a similar test for MVStoreChatHistory?

Reply:
@InAnYan Please answer my question here :)

Reply:
Now, I made tests for every class, that needs persistence